db/migrate/20230407170430_add_payload_credentials_configuration_scripts.rb

```diff
@@ -0,0 +1,12 @@
+class AddPayloadCredentialsConfigurationScripts < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :configuration_scripts, :miq_group,   :type => :bigint, :index => true
+    add_column    :configuration_scripts, :userid,      :string
+    add_column    :configuration_scripts, :payload,     :jsonb
+    add_column    :configuration_scripts, :credentials, :jsonb
+    add_column    :configuration_scripts, :context,     :jsonb
+    add_column    :configuration_scripts, :output,      :jsonb
+    add_column    :configuration_scripts, :status,      :string
+    add_reference :configuration_scripts, :miq_task,    :type => :bigint, :index => true
```

Comment:
```suggestion
    change_table :configuration_scripts |t|
      t.references :miq_group
      t.string     :userid
      t.jsonb      :payload
      t.jsonb      :credentials
      t.jsonb      :context
      t.jsonb      :output
      t.string     :status
      t.references :miq_task
    end
```

Pretty sure `references` defaults to `:type => :bigint`, and `:index => true`
It does on 7.x, probably on 6.x

Reply:
I was unsure if we wanted `user_id` or `userid`, but we seem to use both everywhere, so not sure it makes a difference.

Reply:
userid is the string, user_id is the integer - in this case we want the string.